setup.py

```diff
@@ -304,6 +304,8 @@ def init_settings_from_config(self):
             pjoin('backend', 'cython'),
             'devices',
         )]
+        if sys.platform.startswith('win'):
+            settings['include_dirs'].insert(0, 'patch')
```

Comment:
>  I guess libzmq also depends on <stdint.h> being available to client applications...

That seems improbable, since it builds successfully without it in master.